test/runner/requirements/constraints.txt

```diff
@@ -11,3 +11,6 @@ ncclient >= 0.5.2 # Need features added in 0.5.2 and greater
 idna < 2.6 # requests requires idna < 2.6, but cryptography will cause the latest version to be installed instead
 paramiko < 2.4.0 ; python_version < '2.7' # paramiko 2.4.0 drops support for python 2.6
 pytest < 3.3.0 ; python_version < '2.7' # pytest 3.3.0 drops support for python 2.6
+ntlm-auth >= 1.0.6 # message encryption support
+requests-ntlm >= 1.1.0 # message encryption support
+requests-credssp >= 0.1.0 # message encryption support
```

Comment:
Do we need, are we planning on adding support for testing CredSSP?

Reply:
Yeah, I wanted to get it in there (and in the container); we can add these to the connection tests. CredSSP is slow enough to negotiate that I don't think we want to run the whole suite on it, though.

Reply:
Cool sounds good.